### PR TITLE
Navigation: Remove duplicated pillars, align pillars with columns

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

- Hides duplicated pillars
- Aligns pillar links with columns
- Makes all menu-items one liner

No animation - just the raw open effect, which should work in all browsers. On top of this we can build fancy stuff ...

## What is the value of this and can you measure success?

New navigation.

## Does this affect other platforms - Amp, Apps, etc?

No.

## Screenshots

(takes a bit of time, huuge gif 🙀, but performance, performance!)

kapture 2017-06-08 at 13 56 27

## Tested in CODE?

No.

### CODE REVIEW

File: static/src/stylesheets/layout/new-header/_menu-item.scss [hunk 1]
```diff
@@ -78,7 +81,10 @@
 
     @include mq(desktop) {
         font-size: 17px;
-        padding-left: 0;
+        overflow-x: hidden;
+        padding: 6px 0;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_menu-item.scss]: $gs-baseline / 2?

**Review by NataliaLKB — APPROVED** [review]
Looks good!

**Comment by NataliaLKB** [comment]
In a separate PR we should make sure we bring back some mobile functionality. In particular:

* Making sure to close all open links/reset scroll position when the menu is closed 
* We used to have a lovely degradation of the design (shown here: https://github.com/guardian/frontend/pull/14019) for small screens. We should bring it back

**Comment by paperboyo** [comment]
> Aligns pillar links with columns

👏 Exactly what I was dreaming about but was afraid to ask for 👏

**Pull request merged**